iaso/api/setup_account.py

```diff
@@ -52,6 +51,26 @@ def create(self, validated_data):
             user_manual_path=validated_data.get("user_manual_path"),
         )
 
+        # Create a setup_account project with an app_id represented by the account name
+        app_id = (
+            validated_data["account_name"]
+            if " " not in validated_data["account_name"]
+            else validated_data["account_name"].replace(" ", "-")
+        )
+        initial_project = Project.objects.create(
+            name=validated_data["account_name"] + " project", account=account, app_id=app_id
+        )
+
+        # Create an initial orgUnit type and link it to project
+        initial_orgunit_type = OrgUnitType.objects.create(name="Country", short_name="country", depth=1)
```

Comment:
I wonder if depth shouldn't be `0` here.
@madewulf, any idea on this?

Reply:
@bmonjoie I checked in the DB how the depth is normally used and I found that it's like orgUnit levels as for Country we have 1, Region there is 2  and district it's  3